src/lib/stitching/logLinkMiddleware.ts

```diff
@@ -0,0 +1,26 @@
+import { ApolloLink } from "apollo-link"
+import { print } from "graphql/language"
+
+const shouldLogLinkTraffic = !!process.env.LOG_HTTP_LINKS
```

Comment:
nice! so the default here would be `false`, right?

Reply:
It's an optional env var, so the default is for it to not exist, which fails the not not